index.html

```diff
@@ -1,57 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta name="description" content="CloudNV takes your picture of a cloud and tells you its type.">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta charset="UTF-8">
-    <title>CloudNV</title>
-    <link rel="stylesheet" type="text/css" href="assets/css/reset.css">
```

Comment:
What is your reasoning for using reset.css? Bootstrap does already have normalize.css, but there are slight differences between them.